js/map.js

```diff
@@ -73,45 +71,62 @@ function getTooltip(props){
       <thead>
         <tr>
           <th>Category</th>
-          <th><span class='float-end'>kW</span></th>
-          <th><span class='float-end'>Projects</span></th>
+          <th><span class='float-end'>Energized</span></th>
+          <th><span class='float-end'>#</span></th>
+          <th><span class='float-end'>Planned</span></th>
+          <th><span class='float-end'>#</span></th>
         </tr>
       </thead>
       <tbody>
         <tr>
           <td>Total</td>
           <td><span class='float-end'>${props.total_kw.toLocaleString()}kW</span></td>
           <td><span class='float-end'>${props.total_count.toLocaleString()}</span></td>
+          <td><span class='float-end'>${props.planned_total_kw.toLocaleString()}kW</span></td>
```

Comment:
A space before `kw` would make these all a lot readable in my opinion, e.g. 49,038kW vs 49,038 kW.

Reply:
good call

Reply:
So it looks like you only did this on the total, I meant it for the whole table 😆 

I'll leave that for later though

Reply:
ah woops - i'll get that fixed